dracula-theme.el

```diff
@@ -370,6 +370,23 @@ read it before opening a new issue about your will.")
                (js3-instance-member-face :foreground ,dracula-cyan)
                (js3-jsdoc-tag-face :foreground ,dracula-pink)
                (js3-warning-face :underline ,dracula-pink)
+               ;; lsp
+               (lsp-ui-peek-peek :background ,dracula-bg)
+               (lsp-ui-peek-list :background ,bg2)
+               (lsp-ui-peek-filename :foreground ,dracula-pink :weight bold)
+               (lsp-ui-peek-line-number :foreground ,dracula-fg)
+               (lsp-ui-peek-highlight :background ,bg4 :foreground ,fg4
+                                      :distant-foreground ,dracula-bg)
```

Comment:
Is there a reason why you don't inherit from the default `highlight` face here? (bg3, fg3)

Reply:
No special reason. I didn't know that there was `highlight` face already defined. I made a change :smile:

Reply:
No problem :) As for `match` bellow, I think it's better to stay homogeneous when possible. But as I'm not a user of this package, some assumption may be wrong :)